ignite/src/main/java/com/yahoo/ycsb/db/IgniteYCSBClient.java

```diff
@@ -0,0 +1,181 @@
+package com.yahoo.ycsb.db;
+
+import com.yahoo.ycsb.ByteIterator;
+import com.yahoo.ycsb.DBException;
+import com.yahoo.ycsb.Status;
+import com.yahoo.ycsb.StringByteIterator;
+import com.yahoo.ycsb.workloads.CoreWorkload;
+import org.apache.ignite.Ignite;
+import org.apache.ignite.IgniteCache;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+
+/**
+ *
+ */
+public class IgniteYCSBClient extends com.yahoo.ycsb.DB {
+
+  public static final String INSERT_ASYNC = "insertAsync";
+  public static final String IGNITE_IPS = "igniteIPs";
+  public static final String BATCH_SIZE = "batchSize";
+
+  private static final Logger LOG = LoggerFactory.getLogger(IgniteYCSBClient.class);
+  private static Ignite ignite;
+  private static IgniteCache<String, Map<String, String>> cache;
+
+  private boolean asyncInsert = false;
+  private int batchSize = 1;
+  private int batchCount = 0;
+  private Map<String, Map<String, String>> batchBuffer = new HashMap();
+
+
+  static synchronized void createIgniteClient(String cacheName, String igniteIPs) {
+    if (ignite == null) {
+      System.out.println("IgniteYCSBClient.createIgniteClient");
+      ignite = IgniteClient.startIgnite(igniteIPs);
+      cache = ignite.getOrCreateCache(cacheName);
+    }
+  }
+
+
+  @Override
+  public void init() throws DBException {
+    System.out.println("Init this.toString() = " + this.toString());
```

Comment:
why Sysout when you have a Logger?

Reply:
my bad. started implementation based on jdbc binding because I wanted to create table on the fly. Those printlns were all over that place. 
Will fix.